app/models/user.rb

```diff
@@ -2,6 +2,7 @@ class User < ApplicationRecord
   validates :email, presence: true, uniqueness: true
   validates :password, presence: { require: true }
   validates :password, confirmation: true
+  validates :password_confirmation, presence: { require: true }
```

Comment:
I thought the line above was supposed to do what this line does, but it was required to pass a test where the confirmation was left out. I hope I'm not repeating myself here.